src/mongoose_cluster_id.erl

```diff
@@ -122,18 +124,9 @@ set_new_cluster_id(ID, mnesia) ->
 %% Get cluster ID
 -spec get_backend_cluster_id(mongoose_backend()) -> maybe_cluster_id().
 get_backend_cluster_id(rdbms) ->
-    SQLQuery = [<<"SELECT v FROM mongoose_cluster_id WHERE k='cluster_id'">>],
-    try mongoose_rdbms:sql_query(global, SQLQuery) of
+    case mongoose_rdbms:execute_successfully(global, cluster_select, []) of
```

Comment:
Could you call `mongoose_rdbms:execute` here to prevent crashing in case of DB errors? (see the comment above).

Reply:
Like above.